src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/JSON1001.SyntaxError.sarif

```diff
@@ -5,7 +5,8 @@
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner"
+        "name": "CodeScanner",
+        "version": "1.0.0"
```

Comment:
No need for this. This is file is invalid JSON (see the comma on line 3).

Reply:
oh! nice :) Thanks


---
In reply to: [434767894](https://github.com/microsoft/sarif-sdk/pull/1899#discussion_r434767894) [](ancestors = 434767894)